### DISCUSSION TITLE
Translation fo the theme

### DISCUSSION THREAD

**Discussion by OxNinja** [pradyunsg/furo#429] [Questions] [answered]
Hello, is this possible to add some translations to the theme?

For example I built my documentation in french and I have this text for the research bar: 
![image](https://user-images.githubusercontent.com/57400445/165059086-a3511df7-6b4c-49f8-922b-7960f69575b0.png)

But for the bottom I have this:
![image](https://user-images.githubusercontent.com/57400445/165059205-49df7790-7b9d-4427-87e0-a126b2ecf562.png)

(`Previous` or `Next` instead of `Précédent` or `Suivant`)

Is this a Sphinx-related issue?

**Comment by pradyunsg** [comment]
Hmm... I think this might be because the translations "database" that ships with Sphinx does not include a translation for "Previous" and "Next". I'll need to investigate this more though.

**Comment by damian-krawczyk** [comment]
Hi @OxNinja
I achieved this by copying furo's `page.html` to `docs/_templates` of my [sphinx-internationalization-test](https://github.com/damian-krawczyk/sphinx-internationalization-test) documentation:

`wget -P docs/_templates https://raw.githubusercontent.com/pradyunsg/furo/main/src/furo/theme/furo/page.html`

and then by generating `*.po` files again, but this time with `sphinx.po`, which lets me translate Previous/Next buttons:

https://github.com/damian-krawczyk/sphinx-internationalization-test/blob/1ded8dbb1b89ec65bfb552b0b805cccc4457208f/docs/locale/pl/LC_MESSAGES/sphinx.po#L25-L31

<img width="166" alt="image" src="https://user-images.githubusercontent.com/9287709/172903238-f8722e0b-6b70-473b-aa43-827f8349b067.png">

**Comment by pradyunsg** [comment, marked as answer]
I've created a tracking issue for this in https://github.com/pradyunsg/furo/issues/434.
